app/(playground)/p/[agentId]/canary/utils.ts

```diff
@@ -169,3 +169,138 @@ export function langfuseModel(llm: TextGenerateActionContent["llm"]) {
 	const [_, model] = llm.split(":");
 	return model;
 }
+
+/**
+ * Formats a timestamp number into common English date string formats
+ */
+export const formatTimestamp = {
```

Comment:
imo: When the processing of time becomes more complex or internationalization (i18n) needs increase, I’d like to migrate to something like Day.js.

[https://day.js.org/](https://day.js.org/)